### ISSUE TITLE
VoiceOver does not read incoming chats

### ISSUE THREAD

**Issue by rkingett** [tigase/siskin-im#166]
**Describe the bug**

VoiceOver, Apple's built in screen reader, should announce incoming chats automatically if a chat is focused.

There is also an issue where the most recent chat message can't gain focus if the history is past 20 messages, but this one would be an improvement worth prioritizing.

At the moment, incoming messages do not get read automatically when a conversation is open. It would vastly improve the accessibility of the app if VO could read incoming chats automatically.

I'm not sure which code allows VoiceOver to read incoming chats automatically like in iMessage, but [this VoiceOver resource should be a good starting point.](https://developer.apple.com/documentation/accessibility/supporting_voiceover_in_your_app)

[VoiceOver reading resource videos](https://developer.apple.com/videos/play/wwdc2019/248/)

[SwiftUI Accessibility: Beyond the basics](https://developer.apple.com/videos/play/wwdc2021/10119/)

**To Reproduce**

1. Enable VoiceOver from Accessibility settings, in the settings app.
2. Open Siskin app.
3. Open a chat conversation
4. wait for an incoming message with VoiceOver turned on.

**Expected behavior**

VoiceOver should read all incoming chats automatically when a chat conversation is open.

**Details (please complete the following information):**
 

- Siskin Version: 7.0.1
- - iPhone model SE 2020.
- - iOS version, 15.3

**Comment by jmdaweb** [comment]
Three years and some months later, this problem still happens. I have started using this app recently. I'm going to open two additional VoiceOver-related issues.
